packages/magic/README.md

```diff
@@ -28,7 +29,7 @@ type magicOptions = {
 import Onboard from '@web3-onboard/core'
 import magicModule from '@web3-onboard/magic'
 
-const magic = magicModule({ apiKey: 'API_KEY' })
+const magic = magicModule({ apiKey: 'API_KEY', userEmail: 'test@email.com' })
```

Comment:
```suggestion
const magic = magicModule({ apiKey: 'API_KEY', userEmail: localStorage.getItem('magicUserEmail') })
```
See below suggestion about updating validation so that the module can be initialized nice and easy like this.